### PR TITLE
Master misc ged

### PR DESCRIPTION
**Pull request by ged-odoo**

### CODE REVIEW

File: src/qweb/qweb.ts [hunk 1]
```diff
@@ -750,7 +760,7 @@ export class QWeb extends EventBus {
             name = '"' + name + '"';
           }
           attrs.push(`${name}: _${attID}`);
-          handleProperties(name, attID);
+          handleProperties(name, "_" + attID);
```
Comment (aab-odoo) [inline src/qweb/qweb.ts:781]: any reason why not to write
```js
`_${attID}`
```
as above?

File: tests/component/slots.test.ts [hunk 2]
```diff
@@ -325,6 +325,70 @@ describe("t-slot directive", () => {
     expect(QWeb.slots["1_default"].toString()).toMatchSnapshot();
   });
 
+  test("slots in t-foreach in t-foreach", async () => {
+    class HOC extends Component {
```
Comment (aab-odoo) [inline tests/component/slots.test.ts:329]: HOC?

File: tests/component/slots.test.ts [hunk 3]
```diff
@@ -325,6 +325,70 @@ describe("t-slot directive", () => {
...
+      static template = xml`
+        <div t-name="HOC"><t t-slot="default" /></div>
+      `;
+    }
+
+    class App extends Component {
+      static template = xml`
+        <div>
+        <t t-foreach="tree" t-as="node1" t-key="node1.key">
```
Comment (aab-odoo) [inline tests/component/slots.test.ts:338]: indentation

**Review by aab-odoo — APPROVED** [review]

**Comment by ged-odoo** [comment]
PR updated

**Review by aab-odoo — APPROVED** [review]

**Pull request merged**